packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -67,6 +73,10 @@ const AccountSwitcher = props => {
         }
     };
 
+    const maxAcountsDisplayed = () => {
+        return props?.account_list?.length > 4;
```

Comment:
It's not a part of this PRs scope but maybe we can breakdown props in this component to avoid calling `props` object every time we want to access the value.

Reply:
Can try to do it here 👍

Reply:
There are more than 50 props. Maybe that's why the props were not broken down 🤔

Reply:
Ok, in that case, no need haha